Dockerfile

```diff
@@ -2,8 +2,14 @@ FROM python:3.6-slim-stretch
 
 ENV PYTHONDONTWRITEBYTECODE=1
 
-# Run everything as olympia user, by default.
-USER olympia
+ARG GROUP_ID=1000
+ARG USER_ID=1000
```

Comment:
These require `--build-arg GROUP_ID=$(id -g) --build-arg USER_ID=$(id -u)` - I'll add docs about that later - I found that we don't have any docs at all about local building so I'll do that together with fixing circleci automatic builds (for local dev) as this may change requirements.